src/app/entities/courses/directives/show-course-status/show-course-status.directive.ts

```diff
@@ -0,0 +1,35 @@
+import { Directive, HostBinding, Input, OnInit } from '@angular/core';
+
+@Directive({
+  selector: '[appShowCourseStatus]',
+})
+export class ShowCourseStatusDirective implements OnInit {
+  FOURTEEN_DAYS_IN_MILLISECONDS = 1000 * 60 * 60 * 24 * 14;
+  COLORS = {
+    BLUE: '#009ECC',
+    GREEN: '#67A300',
+    TRANSPARENT: 'transparent',
+  }
```

Comment:
I believe that these values can be a stand alone constants

Reply:
Yes, I'll convert them to standalone constants